src/mocks/responses/validation-metadata.js

```diff
@@ -0,0 +1,333 @@
+export const validationMetadata = {
```

Comment:
This is only being used in the `validation-results-sidebar.test.js`, so I'd co-locate this with that file. Maybe `validation-metadata.response-dummy.js`. We don't have a convention for where to put this data, I don't even remember having seen test data outside of test files. I wouldn't mind putting this into the `.test.js` file actually, code editors can collapse that code anyway, so it shouldn't be in the way when browsing the test code.

Also I don't think that this is a "mock", although I'm sure there are different definitions for what term means what out there. Referring to [this stackoverflow answer](https://stackoverflow.com/a/17810004), the `validationMetadata` is "dummy" data, if I interpret the answer correctly.

Reply:
yeah that was the structure recommended by msw .. I will change it now that we got rid of it, and put it closer to the test